healthcareai/tests/test_deploy_supervised_model_class.py

```diff
@@ -120,3 +121,22 @@ def runTest(self):
 
     def tearDown(self):
         del self.o
+
+
+class TestValidateDestinationTableConnection(unittest.TestCase):
+    def test_raises_error_on_table_not_existing(self):
+        try:
+            result = validate_destination_table_connection('localhost', 'foo', 'bar', 'baz')
+            # Fail the test if the above call doesn't throw an error
+            self.fail()
+        except HealthcareAIError as e:
```

Comment:
Great point. My thought there was to exercise the specific types of errors (since there were a few), but I think I agree with you and may have gone a bit overboard w/ testing here.